curve-fi/src/components/SwapPane.jsx

```diff
@@ -18,6 +18,22 @@ function SwapPane() {
     toAmount: 0,
     toCurrency: "USDC",
   });
+  console.log(initialState);
```

Comment:
is this log statement needed

Reply:
Nope, even you commenting on this makes me feel like we are going to make great progress.